src/configuration/index.ts

```diff
@@ -373,4 +377,39 @@ export default class ConfigurationAPI extends WebAPI {
   async getLicenseWebhooksState(client_id?: string): Promise<WebhooksState> {
     return this.send("get_license_webhooks_state", { client_id });
   }
+
+  /**
+   * Creates an auto access data structure, which is a set of conditions for the tracking URL and geolocation of a customer.
+   * @param opts - options for
```

Comment:
something is missing in the description?

Reply:
Fixed. Completed the description for parameters.